docs/fundamentals/executor/hub/hub-portal.md

```diff
@@ -0,0 +1,53 @@
+# Hub Portal
+
+Jina Hub is a marketplace for Executors. Here you can upload the Executors you have created or use the ones already developed by the community. If it's the first time you're developing your Executor you can start by checking our {ref}`tutorials <create-hub-executor>` that will guide you through the process.
+ 
+Let's see the [Hub portal](https://hub.jina.ai) in detail.
+
+## Catalog page
+
+The main page contains a list of all Executors created by Jina developers all over the world. By default, you will see the Editors Pick first at the top of the list. Those are the Executors that were Highlighted by the Jina team. 
+
+```{figure} ../../../../.github/hub-website-list.png
+:align: center
+```
+
+You can sort the list of displayed Executors by *Trending* and *Recent* using the drop-down menu on top. Otherwise, if you want to search for a specific Executor, you can use the search box on the top or use the tags for specific keywords such as Image, Video, TensorFlow, and so on:
```

Comment:
```suggestion
You can sort the list of Executors by *Trending* and *Recent* using the drop-down menu on top. Otherwise, if you want to search for a specific Executor, you can use the search box at the top or use tags for specific keywords like Image, Video, TensorFlow, and so on:
```